spec/graphql/queries/show_event_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Events' do
+   
+  it "queries and returns all events" do
+    e = create(:event, name: 'FirstEvent')
+    e2 = create(:event, name: 'AnotherEvent')
+    
+    query_string = <<-GRAPHQL
+    query
+     {
+       events {
+          id
+          duration
+          name
+        }
+      }
+      GRAPHQL
+      
+    result = WebsiteOneBackendApiSchema.execute(query_string)["data"]["events"]
+  
+    names = result.collect {|x| x["name"]}
+
+    assert names.detect { |e| e == 'FirstEvent' }
+    assert names.detect { |e| e == 'AnotherEvent' }
```

Comment:
Is assert provided by RSpec? It looks like a minitest assertion.

It would be nice to use RSpec expectations here.

Reply:
We tried using `expect(array_one).to match(array_two)` and `assert array_one =~ array_two` plus a couple more but they were just not working, so Matt made a this work around, I guess we will be looking into it more. Maybe they don't work easy with GQL? not sure...

Reply:
I put those lines in a custom rspec matcher, which may be the best I can do.  Does that look better?